docs/observability/reliability-management-slo/create-slo.md

```diff
@@ -118,19 +118,45 @@ Follow the instructions below based on the query type:
 10. Click **Save**. To create a monitor, click [Save and Create Monitor](#creating-slo-monitors).
 
 
-## Creating SLOs from Logs, Metrics, Monitors
+## Create a Log-based SLO
 
 You can create SLOs directly from your Sumo Logic log search. This allows you to validate queries, quickly create SLOs, and re-use queries from existing dashboard panels.
 
 1. Enter a new **Log search** query (or use an existing one).<br/><img src={useBaseUrl('img/observability/log-search.png')} alt="log search" />
 1. Click the **More Actions** (kebab icon) dropdown menu.<br/><img src={useBaseUrl('img/observability/slo-more-actions-kebab.png')} alt="More Actions" />
 1. Click **Create an SLO**.<br/><img src={useBaseUrl('img/observability/slo-create.png')} alt="Create an SLO" width="200"/>
 
+
+## Create a Monitor-based SLO
+
+Critical Monitors are good candidates to convert to SLOs. As an example, if you have an existing Monitor that's set to alert you to Critical conditions. For example, in Kubernetes env, more than 5 pods in terminated state in 5-minute interval.
+
+From the Monitor UI, you can create a window-based SLO for a given trigger condition (like Critical) given time period 1m window duration. To create a Monitor-based SLO:
+
+:::important
+Your Monitor must be in an active state prior to creating a Monitor-based SLO.
+:::
+
+1. Open the alert panel > **More Actions** > **Create SLO**.
+1. In the **New SLO** window, you'll see that your Monitor conditions have auto-populated. You're welcome to revise and define additional SLI and SLO parameters here.
+1. When you're done, click **Save** (what does "Save and create Monitor do"?)
```

Comment:
> what does "Save and create Monitor do"?

This creates a new monitor based SLO behind the scene and takes user quickly to define a new monitor on top of that monitor based SLO. You can assume it like a quick create monitor on top of monitor based SLO.

Reply:
Thank you!